nirc_ehr/resources/web/nirc_ehr/model/sources/DeathNecropsy.js

```diff
@@ -0,0 +1,152 @@
+EHR.model.DataModelManager.registerMetadata('DeathNecropsy', {
+    byQuery: {
+        'study.deaths': {
```

Comment:
Disposition in the death section is not populating the drop down for me. Do we have that data?

Reply:
we do not

Reply:
Wendy has added these to the recent spec. Now added.